cover_agent/AICaller.py

```diff
@@ -45,6 +45,7 @@ def call_model(self, prompt: dict, max_tokens=4096):
                 messages=messages,
                 max_tokens=max_tokens,
                 stream=True,
+                temperature=0.2, # low, but not zero. This is a good default for most cases
```

Comment:
The Test-LLM paper called out that a temperature of 0 was the best (statistically speaking). I also found it to provide the best results. What is the reason for changing it to 0.2?